schema/schema.json

```diff
@@ -4,206 +4,300 @@
+                  "type": ["string","null"],
+                  "format": "date-time",
+                  "description": "The date and time about when any part of the information for this deal was last updated. Should be in yyyy-mm-ddThh:mm:ssZ format."
+              },
+              "dataSource": {
+                "title": "Data Source",
+                "type": ["string","null"],
+                "format": "uri",
+                "description": "A web link pointing to the source of this data, such as the actual data file or the publisher's website."
+              }
+          }
+      },
+      "Grant": {
+          "type": "object",
+          "title": "Grant",
+          "description": "A non-repayable award",
+          "properties": {
+              "identifier": {
+                  "title": "Identifier",
+                  "type": "string"
+              },
+              "currency": {
+                  "title": "Currency",
+                  "type": "string"
+              },
+              "fundingProgramme": {
+                  "title": "Funding Programme",
+                  "description": "Funding programme that the grant forms a part of.",
+                  "$ref": "#/definitions/FundingProgramme"
+              }
+          }
+      },
+      "Equity": {
+          "type": "object",
+          "title": "Equity",
+          "description": "An equity investment",
+          "properties": {
+              "identifier": {
+                  "title": "Identifier",
+                  "type": "string"
+              },
+              "currency": {
+                  "title": "Currency",
+                  "type": "string"
+              },
+              "fundingProgramme": {
+                  "title": "Funding Programme",
+                  "description": "Funding programme that the grant forms a part of.",
+                  "$ref": "#/definitions/FundingProgramme"
+              }
+          }
+      },
+      "Measurement": {
+        "type": "object",
+        "title": "Measurement",
+        "description": "A measurement related to an entity at a point in time.",
+        "properties": {
+          "identifier": {
+            "title": "Identifier",
+            "description": "A unique identifier for the measurement",
+            "type": "string"
+          },
+          "measuredEntityIdentifier": {
+            "title": "Measured entity identifier",
+            "description": "The unique identifier of the measured entity",
+            "type": "string"
+          },
+          "measure": {
+            "title": "Meausre",
+            "description": "The measure and vocabulary it is drawn from.",
+            "$ref": "#/definitions/Classification"
+          },
+          "statistic": {
+            "title": "statistic",
+            "description": "The type of statistic described in the value field",
+            "type": "string",
+            "enum": ["identity", "mean", "median"]
```

Comment:
"identity" as a type of statistic? Not sure what that means? Could add "max", "min", "percent", "ratio", "range", "difference"?